packages/app/src/domain/topical/components/topical-tile/topical-tile.tsx

```diff
@@ -0,0 +1,137 @@
+import { Box } from '~/components/base';
+import css from '@styled-system/css';
+import styled from 'styled-components';
+import { Heading } from '~/components/typography';
+import { LinkWithIcon } from '~/components/link-with-icon';
+import { asResponsiveArray } from '~/style/utils';
+import { colors } from '@corona-dashboard/common';
+import DynamicIcon from '~/components/get-icon-by-name';
+import { Chevron, Down, Up } from '@corona-dashboard/icons';
+import { Markdown } from '~/components/markdown';
+import { TopicalIcon } from '@corona-dashboard/common/src/types';
+import { isDefined } from 'ts-is-present';
+
+interface IconWrapperProps {
+  iconColor: any;
+}
+
+type TrendIcon = {
+  direction: 'UP' | 'DOWN';
+  color: 'GREEN' | 'RED';
+} | null;
+
+type Cta = {
+  label: string;
+  href: string;
+} | null;
+
+interface TopicalTileProps {
+  title: string;
+  tileIcon: TopicalIcon;
+  trendIcon?: TrendIcon;
+  dynamicDescription: string;
+  cta?: Cta;
```

Comment:
```suggestion
  cta: Cta;
```
The call to action prop is always there, but it can be `null`. So it doesn't need to be optional :-)

Reply:
That's correct, but I imagine it could be optional in some cases when used outside the current JSON and therefore making it optional. How about the trendIcon? That one is optional as well.